contracts/ERC721MLite.sol

```diff
@@ -0,0 +1,571 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import "@openzeppelin/contracts/utils/cryptography/SignatureChecker.sol";
+import "erc721a/contracts/extensions/ERC721AQueryable.sol";
+import "./IERC721M.sol";
+
+/**
+ * @title ERC721MLite
```

Comment:
what's the difference between the non-lite version?

Reply:
1. Removed `crossmint` function
2. Removed / mocked out functions that not used by our FE